src/status_im/wallet/accounts/core.cljs

```diff
@@ -157,8 +160,11 @@
   (let [error (:error (types/json->clj phrase-warnings))]
     (if-not (string/blank? error)
       (new-account-error cofx :account-error error)
-      {::import-account-seed {:passphrase      passphrase
-                              :hashed-password hashed-password}})))
+      (let [{:keys [db]} cofx
+            accounts (:multiaccount/accounts db)]
+        {::import-account-seed {:passphrase      passphrase
+                                :hashed-password hashed-password
+                                :accounts accounts }}))))
```

Comment:
a perfectionist is crying :)

Reply:
As mentioned in the Review Notes, I'm very happy to iterate over this and change it to make it more reframe-y. Thoughts?

Reply:
i mean formatting, and btw lint has failed